### PR TITLE
[wip] - Feature; send alerts on disabled credentials

### PR DESCRIPTION
**Pull request by lsh-0**

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -280,15 +282,45 @@ def send_email(to_addr, subject, content):
     }
     return ses.send_email(**kwargs)
 
+def email_user__old_credentials_disabled(user_csvrow):
```
Comment (giorgiosironi) [inline src/main.py]: two underscores, confusing?

File: src/main.py [hunk 2]
```diff
@@ -280,15 +282,45 @@ def send_email(to_addr, subject, content):
...
+    ensure('results' in user_csvrow, "`email_user__old_credentials_disabled` requires the results of calling `execute_user_report`")
+    ensure('disable' in user_csvrow['results'] and user_csvrow['results']['disable'],
+           "`email_user__old_credentials_disabled` requires a key was successfully disabled")
+    disabled_credential_key = dict(user_csvrow['actions'])['disable']
+    name, to_addr = vals(user_csvrow, 'name', 'email')
+    subject = 'AWS credentials disabled'
+    content = '''Hello {insert-name-of-human},
+
+Your old credentials "{insert-disabled-credential-key}" have exited their grace period
```
Comment (giorgiosironi) [inline src/main.py]: missing `.`

File: src/main.py [hunk 3]
```diff
@@ -280,15 +282,45 @@ def send_email(to_addr, subject, content):
...
+
+{insert-grace-period} or more days ago you were issued new AWS credentials to replace
+your old set "{insert-disabled-credential-key}". They have now been disabled.
```
Comment (giorgiosironi) [inline src/main.py]: could state the subject of the original email, so that it's clear what to look for in an inbox.

File: src/main.py (hunk 1 above)
Comment (lsh-0) [reply src/main.py]: are you confused?

Comment (lsh-0) [reply src/main.py]: Sorry, that was catty. I think it's perfectly clear.

File: src/main.py (hunk 3 above)
Comment (lsh-0) [reply src/main.py]: it's a wip, I hadn't finished writing the message template. There are actually two attempts at the message here.

File: src/main.py (hunk 1 above)
Comment (giorgiosironi) [reply src/main.py]: Double underscores in Python have special meaning for name mangling and/or magic methods like `__repr__`, that's why I classify an ordinary method using them as confusing (as you would if there was a method named `EmailUserOldCredentialsDisabled`; why it's using a class-name convention?)

Comment (lsh-0) [reply src/main.py]: It's a function, not an 'ordinary method'. This isn't a problem to be 'fixed' but you can always open a PR

**Comment by lsh-0** [comment]
I think I've told you off before about reviewing stuff before I've added you as a reviewer. I even prefixed it with 'wip'. It's *really* irritating and makes me irrationally adversarial to anything you suggest. I don't care how other devs might feel about it, but you're not helping me by jumping in early.

**Comment by giorgiosironi** [comment]
Ah sorry for getting here early, I followed the link in https://github.com/elifesciences/issues/issues/4323#issuecomment-421268853 because I thought this was required to be completed to disable the credentials; but they are disabled already now so it probably isn't.

**Comment by lsh-0** [comment]
I jump across tickets and put those links there so I remember where I left off. Sorry for snapping at you.

**Pull request merged**